warehouse/query-core/src/test/java/datawave/query/function/TLDScanRangeProviderTest.java

```diff
@@ -0,0 +1,43 @@
+package datawave.query.function;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Range;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TLDScanRangeProviderTest {
+    
+    private final Key docKey = new Key("row", "datatype\0d8zay2.-3pnndm.-anolok");
```

Comment:
add a child example?

Reply:
Added child and grandchild examples by default